conjure-java-jersey-server/src/test/java/com/palantir/conjure/java/server/jersey/AuthTest.java

```diff
@@ -195,6 +234,11 @@ public String getAuthHeader(AuthHeader value) {
             return value.toString();
         }
 
+        @Override
+        public String getAuthHeaderNullable(@Nullable AuthHeader value) {
+            return value == null ? "" : value.toString();
```

Comment:
could we use a descriptive value instead of empty string to denote the header was null?

Reply:
Good call